frontend/src/metabase/lib/embed.js

```diff
@@ -47,10 +47,16 @@ export function initializeEmbedding(store) {
 }
 
 function sendMessage(message) {
-  window.parent.postMessage(
-    { metabase: message },
-    MetabaseSettings.get("embedding-app-origin"),
-  );
+  // Since postMessage does not support multiple origins and sending message to "*"
+  // is discouraged, what we can do is send a message to each origin and messages
+  // where the origin does not match will be dropped by the browser.
+  const origins = MetabaseSettings.get("embedding-app-origin").split(" ");
```

Comment:
I see in `public_settings.clj` that `embedding-app-origin` has no default value, so let's make sure we have a string here by doing something like:
```
const origins = (MetabaseSettings.get("embedding-app-origin") || "").split(" ");
```